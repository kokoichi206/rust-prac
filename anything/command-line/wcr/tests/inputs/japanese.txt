日本語のテキストです。
しかも複数行ありますぞ。
