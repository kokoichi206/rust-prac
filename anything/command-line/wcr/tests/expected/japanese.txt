	2	2	71 tests/inputs/japanese.txt
